docs/maintainer-notes.md

```diff
@@ -32,3 +32,35 @@ Or regenerate everything:
 ```cli
 ./generate --all
 ```
+
+## Creating a new client
+
+If you want to create a new AWS client, there are a few steps to take. Below
+is an example for DynamoDB.
+
+1. Clone the async-aws/aws repository.
+1. Run `composer install`
+1. Create `./src/DynamoDb/DynamoDbClient.php` and add an empty `DynamoDbClient` class.
```

Comment:
Whats's about `.gitattributes`, `.gitignore`, `LICENSE`, ... files?

I suggest to create a `.skeleton` directory that contains everything, et let user rename files. What do you think?

Reply:
Yeah. Adding skeleon makes sense. I didnt include this because that is simple to ask later in the PR review.